src/components/PageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+type Props = {
+  title: string;
+};
+
+export const PageTitle: React.FC<Props> = React.memo(
+  ({ title }) => (
+    <h1 className="title">{ title }</h1>
+  ),
+);
```

Comment:
I don't think you should really need this component, because you use only title here and I suppose this component won't change in any case

Reply:
I agree with @Fynkzia